pages/accountLists/[accountListId]/coaches.page.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react';
+import Head from 'next/head';
+import { useTranslation } from 'react-i18next';
+import { CoachingList } from 'src/components/Coaching/CoachingList';
+
+const CoachesPage = (): ReactElement => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Head>
+        <title>MPDX | {t('Coaches')}</title>
```

Comment:
```suggestion
        <title>MPDX | {t('Coaching Accounts')}</title>
```
(It's what the current app has)

I think `Coaching` could work too.